## Урок 5. Применение математической статистики для проверки гипотез в реальной жизни для популярных метрик
### 1)Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%.
Дайте краткие рекомендации команде .Результаты: 
- 1) Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550 . 
- 2) Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228 
- 3) Конверсии : conv1 = 1.05% conv2 = 1.47% .

In [4]:
import math

# Исследуем кол-ные данные, которые подчиняются закону нормального распределения
# Сравниваем с заданным значением, 1 группу
# стало быть используем Z-тест

n1 = 15550
n2 = 15550
N1 = 164
N2 = 228
conv1 = N1 / n1
conv2 = N2 / n2
# расчитываем стандартное отклонение
SD1 = math.sqrt(conv1 * (1 - conv1))
SD2 = math.sqrt(conv2 * (1 - conv2))

Z = (conv2 - conv1) / math.sqrt(SD1 ** 2 / n1 + SD2 ** 2 / n2)

# критическое значение
alpha = 0.05
crit_value = 1.96


Z


3.253608320772605

3.253608320772605  > 1.96 <br>
Z > crit_value <br> - это означает что есть статистически значимые различия. 


### 2) Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . 
- Какой статистический критерий тут лучше всего подойдёт и почему ?

 ответ - наилучший метод чтобы проверить статистическую значимость изменения для пропорций это метод Хи‑Квадрат.